### ISSUE TITLE
Feature request: Load texture atlases with same frame names

### ISSUE THREAD

**Issue by vpmedia** [GoodBoyDigital/pixi.js#1715]
Hi, I've re-created my previous request for Pixi.JS v3:
https://github.com/GoodBoyDigital/pixi.js/issues/861

And it is well described here:
http://www.html5gamedevs.com/topic/5445-best-way-to-load-texture-atlases-with-the-same-frame-names-possible-pr/

My small patch to make my things work:

Line 23682:

    // lets also add the frame to pixi's global cache for fromFrame and fromImage fucntions
    core.utils.TextureCache[route + '/' + resource.data.meta.image + '/' + i] = resource.textures[i];

**Comment by englercj** [comment]
While this does make you able to use `.fromImage()` and `.fromFrame()` with multiple matching frame names, what it also does is force you use the url in `.fromFrame()` which kind of defeats the purpose.

In the past we've been OK with collision here because it is basically a sugar function, something easy to get running immediately but we expected people to be using their resources directly for most more complicated things. That is, using the `resources` object after loading for creating all your sprites instead of relying on the weak internal cache methods.

If we make users put the full URL for .fromFrame, then it is easier to just use resources and at that point there is no reason for the function to exist at all.

So basically my question to you is, why *must* you use the convenience functions instead of just using the resources object you get from the loader to do all your creation?

**Comment by vpmedia** [comment]
I can do it using the resources object, thanks for clarifying the concept.
I did not found any migration guide / complex example using the loader.
Thanks.
